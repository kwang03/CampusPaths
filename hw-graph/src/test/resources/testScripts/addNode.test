#Tests the ability to add nodes to a graph

#Create graph
CreateGraph myGraph

#Adds nodes and then lists the nodes in the graph
AddNode myGraph n1
ListNodes myGraph

AddNode myGraph n2
ListNodes myGraph

AddNode myGraph n3
ListNodes myGraph

AddNode myGraph n4
ListNodes myGraph

AddNode myGraph n5
ListNodes myGraph