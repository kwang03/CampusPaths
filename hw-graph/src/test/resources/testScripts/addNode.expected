#Tests the ability to add nodes to a graph

#Create graph
created graph myGraph

#Adds nodes and then lists the nodes in the graph
added node n1 to myGraph
myGraph contains: n1

added node n2 to myGraph
myGraph contains: n1 n2

added node n3 to myGraph
myGraph contains: n1 n2 n3

added node n4 to myGraph
myGraph contains: n1 n2 n3 n4

added node n5 to myGraph
myGraph contains: n1 n2 n3 n4 n5

added node n5 to myGraph
myGraph contains: n1 n2 n3 n4 n5